assets/js/googlesitekit/api/index.js

```diff
@@ -230,6 +230,7 @@ export const get = async (
 	datapoint,
 	data,
 	{ cacheTTL = HOUR_IN_SECONDS, useCache = undefined, signal } = {}
+	// eslint-disable-next-line require-await
```

Comment:
Doesn't look like this function needs to be `async` as `siteKitRequest` is async itself and returns a Promise.